packages/salesforcedx-utils-vscode/package.json

```diff
@@ -43,7 +43,7 @@
     "vscode": "^1.40.0"
   },
   "scripts": {
-    "compile": "tsc -p ./",
+    "compile": "tsc -p ./ && webpack",
```

Comment:
Doing this so we only webpack the code under the src folder.

Reply:
Does there need to be both a `tsc` and `webpack` call if webpack uses the ts-loader?

Reply:
Yes because I couldn't find a way to configure webpack to not bundle the code under `test` since tsloader uses the tsconfig.json file too. So this allows us to compile the tests with the normal typescript compiler and the code under `src` with tsloader.

Reply:
Gotcha 👍  sounds good.

Reply:
Did we try excluding the test folder? https://stackoverflow.com/questions/52059366/how-to-exclude-directory-from-getting-bundled-by-webpack

Although I think if webpack doesn't bundle the code we still need to get tests compiled

Reply:
Yeah, if you look at the current webpack config I'm not mapping the test code anymore. However, since tsloader uses tsconfig.json it still picks up the test code. I played with having two tsconfigs; one for `src` which would be used by webpack, the other for `test` and used by typescript. That started to get complex real fast so I opted for going this route.